pkg/controller/direct/dataform/repository_mappings.go

```diff
@@ -28,43 +29,99 @@ func DataformRepositorySpec_FromProto(mapCtx *direct.MapContext, in *pb.Reposito
 	out := &krm.DataformRepositorySpec{}
 	out.GitRemoteSettings = RepositoryGitRemoteSettings_FromProto(mapCtx, in.GetGitRemoteSettings())
 	out.WorkspaceCompilationOverrides = RepositoryWorkspaceCompilationOverrides_FromProto(mapCtx, in.GetWorkspaceCompilationOverrides())
+	out.DisplayName = direct.LazyPtr(in.GetDisplayName())
+	out.GitRemoteSettings = RepositoryGitRemoteSettings_FromProto(mapCtx, in.GetGitRemoteSettings())
```

Comment:
Yeah, it looks like you're already calling  RepositoryGitRemoteSettings_FromProto so I'm confused why you need to call it again!

Reply:
https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/2443#discussion_r1712386454